src/sdk/service/network-validation-service/network-validation-service.ts

```diff
@@ -0,0 +1,468 @@
+import { IPv4AddressRange, IPv4NetworkDescriptor } from './network-validation-interface';
+
+/**
+ * Network Validation Service
+ */
+export abstract class NetworkValidationService {
+  /**
+   * Gets the integer representation of an IP address.
+   * @param {string} ip: the ip address.
+   * @return {number} The integer representation.
```

Comment:
be explicit in your docs about the fact that this method returns null when the input does not represent a valid IPv4 address

Reply:
same comment applies to many of the methods in here